src/librustc_trans/cabi_sparc64.rs

```diff
@@ -24,7 +24,7 @@ fn is_homogeneous_aggregate<'a, 'tcx>(cx: &CodegenCx<'a, 'tcx>, arg: &mut ArgTyp
         let valid_unit = match unit.kind {
             RegKind::Integer => false,
             RegKind::Float => true,
-            RegKind::Vector => arg.layout.size.bits() == 128
+            RegKind::Vector => arg.layout.size.bits() == 256
```

Comment:
Should this be *either* `128` or `256`, or just `256`?

Reply:
On a second look, is this change needed at all? `is_homogeneous_aggregate` refers to a very specific shape of `struct` and this check is for SIMD vectors - which I doubt you are using. The other two changes in this file seem more relevant.